selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -257,6 +258,7 @@ private void captureScreenshot(ITestResult result) {
 
     collectInjectedWebDrivers(testInstance, webDrivers);
     webDrivers.forEach(webDriver -> captureScreenshot(result, webDriver));
+    webDrivers.forEach(webDriver -> dumpHtmlCodeFromTheCurrentPage(result, webDriver));
```

Comment:
Not good name for method for because you not just capture screenshot but basically ok for me